### PR TITLE
Add DeleteCommentDialog

### LINKED ISSUE #2851: UI - create dialog to delete comment
Dialog that opens up when we click delete comment button

Figma spec:

https://www.figma.com/file/fw2pDIUlkhCkJE79ZyTbjp/CoinMachine?node-id=3004%3A24865

The story behind:

#2711

### PR DESCRIPTION
**Pull request by alicjakujawa**
## Description

- This PR adds in DeleteCommentDialog

**New stuff** ✨

* DeleteCommentDialog

Screenshot 2021-11-10 at 08 45 28

Resolves #2851

### CODE REVIEW

**Review by chinins — APPROVED** [review]
All good. It would just be great to have some code sample to open the dialog with proper props in the description.

I would also add the Action form around it straight away as we know it will be used but I guess it can be considered not part of the UI.

File: src/modules/dashboard/components/CoinMachine/DeleteCommentDialog/DeleteCommentDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  },
+});
+
+const displayName = 'dashboard.CoinMachine.DeleteCommentDialog';
+
+interface Props extends DialogProps {
+  comment: CommentProps;
+}
+
+const DeleteCommentDialog = ({ cancel, comment }: Props) => {
```
Comment (rdig) [inline src/modules/dashboard/components/CoinMachine/DeleteCommentDialog/DeleteCommentDialog.tsx:37]: This does not live inside Coin Machine, as it a general, `core` level component.

For that it needs to be both moved and renamed:
- Move it to `src/modules/core`
- Rename it to `CommentDeleteDialog` _(all our other core level comment components have the `Comment` prefix)_

**Review by rdig — APPROVED** [review]
Very nice work! :100: 

However, before merging it please look at the comment I left, and move it / rename it into the `core` module.

Screenshot from 2021-11-10 22-25-22

**Review by ArmandoGraterol — APPROVED** [review]
Looks good to me! Nice work :)

**Pull request merged**